CMakeLists.txt

```diff
@@ -784,6 +784,9 @@ elseif(USING_QT_UI)
       Qt/mainwindow.h
     )
     add_definitions(-DUSING_QT_UI)
+    if(USING_GLES2)
+        add_definitions(-DQT_OPENGL_ES -DQT_OPENGL_ES_2);
```

Comment:
`;` (semicolon) at the end of expression breaks build:
```
CMake Error at CMakeLists.txt:788:
  Parse error.  Expected a command name, got unquoted argument with text ";".
```

Reply:
Derp.

-[Unknown]